.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "src/libmongoc"]
 	path = src/libmongoc
-	url = https://github.com/mongodb/mongo-c-driver.git
+	url = https://github.com/alcaeus/mongo-c-driver.git
```

Comment:
I'll update this PR once all the pull requests in libmongoc have been merged.

Reply:
Yes, the change also includes my work done for CDRIVER-3997, which is ready but was already building on 2 PRs that were in review, so I held off on creating the PR for that.